parser/src/java/org/apache/hadoop/hive/ql/parse/HiveParser.g

```diff
@@ -863,7 +863,7 @@ import org.apache.hadoop.hive.conf.HiveConf;
   }
   protected boolean nullsLast() {
     if(hiveConf == null){
-      return false;
+      return HiveConf.ConfVars.HIVE_DEFAULT_NULLS_LAST.defaultBoolVal;
```

Comment:
Good idea, on keeping this change as separate commit but let's also give it a unique JIRA id since I don't see any change landing in Hive without a JIRA reference.

Reply:
Sure, I have opened a separate PR with a new JIRA ticket: https://github.com/apache/hive/pull/2990